config/application.rb

```diff
@@ -1,6 +1,19 @@
 require_relative 'boot'
 
-require 'rails/all'
+require 'rails'
+# Pick the frameworks you want:
+require 'action_cable/engine'
+require 'action_controller/railtie'
+require 'action_mailbox/engine'
+require 'action_mailer/railtie'
+# require "action_text/engine"
```

Comment:
why do we want to remove action text?

Reply:
Because of this warning

<img width="798" alt="Screenshot 2021-03-15 at 17 29 53" src="https://user-images.githubusercontent.com/33032571/111217226-1c47b100-85b4-11eb-83c3-a3f1e770bf4a.png">

I discussed it on our `ruby` channel and taking into account that is an API the team suggested going with this solution